### ISSUE TITLE
Page / Element - Strange behavior when moving between views

### ISSUE THREAD

**Issue by flukeout** [mozilla/webmaker-android#1982]
Steps to reproduce
* Create a new project
* Edit the start page
* Add a toucan
* Make it huge!!
* Go back to map view
* Page thumbnail is correct
* Edit the page again
* Make the toucan small
* Go back to map view
* Toucan still appears large in page thumbnail

It should reflect the new, smaller size. If you add a new element to the page, then the thumbnail updates, but often to an inaccurate representation of the page.

**Comment by thisandagain** [comment]
I'm seeing this fairly consistently (8/10 tries this morning running latest on Nexus 5 / 5.1.1) throughout all elements, not just images. Marking as critical.

**Comment by k88hudson** [comment]
ok, on it

**Comment by thisandagain** [comment]
I'm still able to reproduce the updating issue, but also seems to cause or amplify another issue in which if I add multiple items it starts to get into a strange loop in which new elements won't be displayed or update. Perhaps a race condition or even an issue where we are using brittle IDs or assigning properties to the wrong element? At the very end of the screen recording you'll notice that a text element just disappears (I did not delete it). Screen recording: https://www.dropbox.com/s/vapc2kysvoid36w/bug.mp4?dl=0

**Comment by Pomax** [comment]
blocked by https://github.com/mozilla/webmaker-android/issues/2009
